Z80Bench.cs

```diff
@@ -6,12 +6,9 @@
 
 namespace z80bench
 {
-    public static class Z80Bench
-    {
-        public static int Main(string[] args)
-        {
-            if (args.Length < 1)
-            {
+    public static class Z80Bench {
```

Comment:
You seem to have inadvertently K&R'd all the curly braces 😱

Reply:
🤦 guess I pushed this accidently. Strange that my rebasing against your comment commit left it in that state though... although maybe I had some small local difference.